lib/coverband.rb

```diff
@@ -90,5 +94,13 @@ def self.runtime_coverage!
     start
     require 'coverband/utils/railtie' if defined? ::Rails::Railtie
     require 'coverband/integrations/resque' if defined? Resque
+    if defined?(Bundler)
```

Comment:
perhaps move into `coverband/integrations/bundler` with the same pattern above?

Reply:
Agreed.

Reply:
Done.